BasicNNMatrix.py

```diff
@@ -88,31 +60,31 @@ def compute_weight_multiplication(weight_list):
         if(i == size-1):
             W = weight_list[i].dot(weight_list[i-1])
         else:
-            print(
-                f"computing dot of {W} with {weight_list[i-1]}")
+            # print(
+            #     f"computing dot of {W} with {weight_list[i-1]}")
             W = W.dot(weight_list[i-1])
 
-    print(f"final weights: {W}")
+    # print(f"final weights: {W}")
     return W
 
 
 def compute_loss_vector(y, z):
     ''' computes the mean squared distances between 2 vectors (y and z) of equal dimensions (1xn)'''
     x = y-z
-    print(f"y is: {y}")
-    print(f"z is: {z}")
-    print(f"x is: {x}")
+    # print(f"y is: {y}")
+    # print(f"z is: {z}")
+    # print(f"x is: {x}")
     squared_vector = np.square(x)
-    print(f"squared error is: {squared_vector}")
-    loss = squared_vector.sum(axis=1)
```

Comment:
`axis=1` was the problem. A `1-D` numpy array does not have axis 1. It only has axis 0. In fact for a 1 D array we don't need to specify the axis to the `sum` function.